README_IT.md

```diff
@@ -527,3 +528,12 @@ Permette di legare i villager al guinzaglio
 - Default value: `true`  
 - Suggested options: `true`, `false`
 - Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`
+
+## playerme
+
+Permette ai giocatori non operatori di eseguire /player su sé stessi
+
+- Type: `boolean`  
+- Default value: `true`  
+- Suggested options: `true`, `false`
+- Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`, `COMMAND`  
```

Comment:
Da posizionare nel punto corretto, al fine che le regole siano disposte in ordine alfabetico

Reply:
da rinominare in commandPlayerme per consistenza